CONTRIBUTING.md

```diff
@@ -86,6 +87,51 @@ assessment.
 
 ## Discussing Mozilla's Position on a Web Specification
 
+### For Mozilla's subject-matter experts
+
+You might be asked for your opinion
+on specifications where you are one of the experts within the Mozilla community,
+or feel like you have valuable contributions to make
+to issues on other specifications even where you haven't explicitly been asked to do so.
+This is the purpose of this repository:
+to help the relevant people in Mozilla's technical leadership come to a conclusion.
+
+The goal of the discussions in this repository is
+to figure out what we think about a specification.
+The key factors to consider are:
+
+* how useful we think the specification is likely to be (or how important the use case it's trying to address is), relative to the likely cost of implementing it and maintaining that implementation
+* whether there are any critical problems we see in the current proposal.
```

Comment:
```suggestion
* whether there are any critical problems we see in the current proposal, particularly around privacy, security, and/or implementability across multiple engines.
```

Reply:
I don't want to push quite that hard towards those areas (since I think it's worth considering others too), so I listed them as examples rather than as "particularly".